src/behaviours/quitting-and-restarting-the-app/opening-application-window-using-tray.test.ts

```diff
@@ -99,12 +105,18 @@ describe("opening application window using tray", () => {
 
       describe("when an application window is reopened using tray", () => {
         beforeEach(() => {
+          focusApplicationMock.mockClear();
```

Comment:
Is this necessary? It is strange that you would have to clear the mocks when you have just created a new instance of it.